idx-template.nix

```
@@ -0,0 +1,23 @@
+{pkgs, branch ? "loic-sharma/decorators", repo ? "https://github.com/loic-sharma/flutter", ...}: {
+    packages = [
+        pkgs.curl
+        pkgs.gnutar
+        pkgs.xz
+        pkgs.git
+        pkgs.busybox
+    ];
+    bootstrap = ''
+        mkdir "$out"
+        cp -a -rf ${./.}/. "$out/"
+        chmod -R +w "$out"
+        rm "$out/idx-template.nix"
+        rm "$out/idx-template.json"
+        rm "$out/dev.nix"
+        mkdir -p "$out/flutter"
+        git clone "${repo}" "$out/flutter/"
```

I think should include the branch:

```suggestion
        git clone -b "${branch}" "${repo}" "$out/flutter/"
```